docs/reference/cluster_manifest.md

```diff
@@ -385,17 +385,25 @@ under the `clone` top-level key and do not affect the already running cluster.
 ## Standby cluster
 
 On startup, an existing `standby` top-level key creates a standby Postgres
-cluster streaming from a remote location. So far streaming from S3 and GCS WAL
-archives is supported.
+cluster streaming from a remote location. Either from a S3 or GCS WAL
+archive or a remote primary. When both of them are set, `standby_host`
+takes precedence.
 
 * **s3_wal_path**
   the url to S3 bucket containing the WAL archive of the remote primary.
-  Optional, but `s3_wal_path`  or `gs_wal_path` is required.
+  Required when the `standby` section is present even when `standby_host` is set.
```

Comment:
Hm, it should not be required if host or gs_wal_path is set. We can work with `oneOf` in the CRD schema validation, I think (see suggestion).